CHANGELOG-1.0.md

```diff
@@ -0,0 +1,121 @@
+# Release notes for master
+
+[Documentation](https://docs.k8s.io/docs/home)
```

Comment:
Should we link to kubernetes-csi docs instead?

Reply:
done.